TFT/src/User/Configuration.h

```diff
@@ -299,11 +299,19 @@
 #define PID_PROCESS_TIMEOUT (15 * 60000)  // (MilliSeconds, 1 minute = 60000 MilliSeconds)
 
 /**
- * M600, M601 ; pause print
+ * M600 ; emulate M600
+ * The TFT intercepts the M600 gcode (filament change) and emulates the logic instead of demanding it to Marlin firmware.
+ *
+ * NOTE: Enable it, in case Marlin firmware does not properly support M600 on the mainboard.
+ */
+#define EMULATE_M600 true  // To enabled: true | To disabled: false (Default: true)
```

Comment:
Shouldn't this be set to `false` by default, so that the introduction of this doesn't unexpectedly chance things for users?

Reply:
that was the default also in the past (when it was hard coded). Not all people have Marlin configured to support real M600.
For the status of the PR, don't worry BTT is awares of what some people is working on